tests/serializers.py

```diff
@@ -170,6 +171,10 @@ class Meta:
         requires=['location.cat_set.*'],
         deferred=True
     )
+    number_of_katz = DynamicMethodField(
+        requires=['location__cat_set'],
```

Comment:
Got it. I thought it might be related to [this recent Django ticket](https://code.djangoproject.com/ticket/25546) but wasn't able to get the test to fail after patching in the fix to that issue.

This is pretty unfortunate.. even with this change, we'd see issues with requests like:

```
GET /users?filter{name}=Foo&include[]=locations.users.cats.
```